shipyard_airflow/plugins/rest_api_plugin.py

```diff
@@ -435,7 +435,6 @@ def index(self):
     # '/api' REST Endpoint where API requests should all come in
     @csrf.exempt  # Exempt the CSRF token
     @expose('/api', methods=["GET", "POST"])
-    @http_token_secure  # On each request,
```

Comment:
Yes, this is not related.  I found a bug when I was testing (had forgotten to remove this line initially and it was causing problems).